packages/ERTP/test/unitTests/mathHelpers/test-natMathHelpers.js

```diff
@@ -61,65 +49,162 @@ test('natMathHelpers', t => {
+    `brand is required in coerce`,
+  );
+});
+
+test('natMathHelpers getValue', t => {
+  t.is(m.getValue(m.make(4n, mockBrand), mockBrand), 4n);
+});
+
+test('natMathHelpers getValue no brand', t => {
+  t.throws(
+    // @ts-ignore
+    () => m.getValue(m.make(4n, mockBrand)),
+    {
+      message:
+        "The brand (an undefined) doesn't look like a brand. Did you pass a value rather than an amount?",
+    },
+    `brand is required in getValue`,
+  );
+});
 
-  // getValue
-  t.is(getValue(make(4)), 4n);
+test('natMathHelpers makeEmpty', t => {
+  const empty = m.make(0n, mockBrand);
 
-  // getEmpty
-  t.deepEqual(getEmpty(), make(0), `empty is 0`);
+  t.deepEqual(m.makeEmpty(MathKind.NAT, mockBrand), empty, `empty is 0`);
+});
 
-  // isEmpty
-  t.assert(isEmpty({ brand: mockBrand, value: 0n }), `isEmpty(0) is true`);
-  t.falsy(isEmpty({ brand: mockBrand, value: 6n }), `isEmpty(6) is false`);
-  t.assert(isEmpty(make(0)), `isEmpty(0) is true`);
-  t.falsy(isEmpty(make(6)), `isEmpty(6) is false`);
+test('natMathHelpers makeEmpty no brand', t => {
   t.throws(
-    () => isEmpty('abc'),
-    { message: /The brand in allegedAmount .* is undefined/ },
+    // @ts-ignore
+    () => m.makeEmpty(MathKind.NAT),
+    {
+      message:
+        "The brand (an undefined) doesn't look like a brand. Did you pass a value rather than an amount?",
+    },
+    `brand is required in coerce`,
+  );
+});
+
+test('natMathHelpers isEmpty', t => {
+  t.assert(m.isEmpty({ brand: mockBrand, value: 0n }), `isEmpty(0) is true`);
+  t.falsy(m.isEmpty({ brand: mockBrand, value: 6n }), `isEmpty(6) is false`);
+  t.assert(m.isEmpty(m.make(0n, mockBrand)), `isEmpty(0) is true`);
+  t.falsy(m.isEmpty(m.make(6n, mockBrand)), `isEmpty(6) is false`);
+  t.throws(
+    () => m.isEmpty('abc'),
+    {
+      message: `The brand in amount (a string) doesn't look like a brand. Did you pass a value rather than an amount?`,
+    },
     `isEmpty('abc') throws because it cannot be coerced`,
   );
   t.throws(
-    () => isEmpty({ brand: mockBrand, value: 'abc' }),
+    () => m.isEmpty({ brand: mockBrand, value: 'abc' }),
     {
-      instanceOf: TypeError,
-      message: 'abc is a string but must be a bigint or a number',
+      message: 'value (a string) must be a Nat or an array',
     },
     `isEmpty('abc') throws because it cannot be coerced`,
   );
   t.throws(
-    () => isEmpty(0),
-    { message: /The brand in allegedAmount .* is undefined/ },
+    () => m.isEmpty(0n),
+    {
+      message: `The brand in amount (a bigint) doesn't look like a brand. Did you pass a value rather than an amount?`,
+    },
     `isEmpty(0) throws because it cannot be coerced`,
   );
+});
 
-  // isGTE
-  t.assert(isGTE(make(5), make(3)), `5 >= 3`);
-  t.assert(isGTE(make(3), make(3)), `3 >= 3`);
+test('natMathHelpers isGTE', t => {
+  t.assert(m.isGTE(m.make(5n, mockBrand), m.make(3n, mockBrand)), `5 >= 3`);
+  t.assert(m.isGTE(m.make(3n, mockBrand), m.make(3n, mockBrand)), `3 >= 3`);
```

Comment:
I'd like to see `m.isGTE(m.make(4n, mockBrand), m.make(3n, otherMockBrand))`. That should throw, right?

Similarly for `isEqual` and `add`.  Ah, I see you did this for `subtract()`

Reply:
Done